	// https://bugs.eclipse.org/bugs/show_bug.cgi?id=111219
	public void test076() throws Exception {
		String source =
			"public class X {\n" + 
			"	void foo() {\n" + 
			"		try {\n" + 
			"			System.out.println(\"Hello\");\n" + 
			"		} catch(Exception e) {\n" + 
			"			e.printStackTrace();\n" + 
			"		}\n" + 
			"	}\n" + 
			"}";
		String expectedOutput =
			"      Exception Table:\n" + 
			"        [pc: 0, pc: 8] -> 11 when : Exception\n";
		checkClassFile("", "X", source, expectedOutput, ClassFileBytesDisassembler.DETAILED | ClassFileBytesDisassembler.COMPACT);
	}

